ingredient_crud/db_config.py

```diff
@@ -5,8 +5,8 @@
 
 # MySQL config
 
-app.config['MYSQL_DATABASE_USER'] = 'localmachine'
-app.config['MYSQL_DATABASE_PASSWORD'] = ''
+app.config['MYSQL_DATABASE_USER'] = 'root'
```

Comment:
no need to commit changes to db_config; it's specific to each developer's machine